supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -246,4 +250,50 @@ class DispatchedConfigAsyncJobExecutionActor(override val standardParams: Standa
   override def killArgs(job: StandardAsyncJob): SharedFileSystemCommand = {
     jobScriptArgs(job, "kill", KillTask)
   }
+
+  protected lazy val exitCodeTimeout: Option[Int] =
+    configurationDescriptor.backendConfig.as[Option[Int]](ExitCodeTimeoutConfig).map(math.abs)
```

Comment:
Why `math.abs`? In that vein it might be good to have minimum acceptable values. 🙂

Reply:
agree, will raise an exception if this is below 0